.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+smoltcp-rs astro
```

Comment:
Shouldn't that be:
```suggestion
* @smoltcp-rs
```
instead?

Reply:
Doesn't it have to be in `@org/team` syntax?

Reply:
Not sure, but as is it's definitely wrong. ;)